kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -16,7 +16,7 @@ pushd $appliance_root
 popd
 
 mkdir -p $app_root
-git clone <%= @manageiq_checkout.remote %> $vmdb_root
+git clone --separate-git-dir /tmp/manageiq.git <%= @manageiq_checkout.remote %> $vmdb_root
```

Comment:
What if we made it conditional on the type of release? A nightly or test build would have it, a release build not.